### PR TITLE
openshift-ansible: add new tests to master using prowgen

### PR DESCRIPTION
**Pull request by vrutkovs**
* Update gcp-major-upgrade test, generated with prowgen
* Add logging on GCP
* Add gluster on AWS

Depends on https://github.com/openshift/ci-operator-prowgen/pull/36

### CODE REVIEW

File: ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-master.yaml [hunk 1]
```diff
@@ -61,3 +61,17 @@ tests:
...
     cluster_profile: gcp-ha
+- as: e2e-gcp-logging
+  commands: TEST_FOCUS=Suite:openshift/conformance/parallel run-tests
+  openshift_ansible:
+    cluster_profile: gcp-logging
+- as: e2e-gluster
+  commands: TEST_FOCUS=Suite:openshift/conformance/parallel/minimal run-tests
+  openshift_ansible_custom:
+    cluster_profile: aws-gluster
+- as: e2e-gcp-major-upgrade-master
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-master.yaml]: You're putting this into the `master` branch configuration so it seems like `-master` suffix is not necessary here

Comment (vrutkovs) [reply ci-operator/config/openshift/openshift-ansible/openshift-openshift-ansible-master.yaml]: Makes sense, done

**Pull request merged**